### PR TITLE
Add auto save local storage

### PR DESCRIPTION
**Pull request by haroldmud**
In this PR I did the following:
- Added the autosave function using javascript

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -143,6 +175,15 @@ function validation() {
...
   errorMessage.style.color = 'green';
+
+  formStore.name = valName;
+  formStore.lastName = valLastName;
+  formStore.email = valEmail;
+  formStore.message = valText;
+
+  localStorage.setItem('formValues', JSON.stringify(formStore));
+  console.log(JSON.stringify(formStore));
+
```
Comment (roshan-bajgain) [inline script.js:186]: - [ ] According to the requirement, When the user changes the content of any input field, the data should be saved to the local storage but In your project, the data will be saved only on the form submitted. So kindly implement the feature to save the data on local storage when the input values are changed. 
```
input.addEventListener('change', () => {
    // your code logic to save data in localstorage
};
```
**Note: Your current implementation of saving data in local storage is working fine, You should implement the same functionality on input change.**

File: script.js [hunk 2]
```diff
@@ -106,6 +106,27 @@ const projectsCards = [
...
 ];
 
+// getting form///////////////////////////////////////////////////////////////////
+
+function getForm() {
+  if (localStorage.length !== 0) {
+    const valuesStored = JSON.parse(
+      localStorage.getItem('formValues'),
+    );
+    console.log(valuesStored);
```
Comment (roshan-bajgain) [inline script.js:116]: - [ ] I would suggest you remove **console.log** statement for clean code

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Your project is beautiful works well and it's almost finished! Check my comments below for the requested changes 👇 

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Hello @haroldmud  😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**